src/Microsoft.Azure.WebJobs.Host/Singleton/SingletonConfiguration.cs

```diff
@@ -8,6 +8,7 @@ namespace Microsoft.Azure.WebJobs.Host
 {
     /// <summary>
     /// Configuration options governing the lock functionality of <see cref="SingletonAttribute"/>.
+    /// These results are passed to <see cref="IDistributedLockManager"/>
```

Comment:
I'll clarify it. 

---
In reply to: [126262845](https://github.com/Azure/azure-webjobs-sdk/pull/1202#discussion_r126262845) [](ancestors = 126262845)